Exceptions/Program.cs

```diff
@@ -0,0 +1,142 @@
+﻿using System;
+
+namespace Exceptions
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //1
+            const int count = 5;
+
+            Worker[] workers = new Worker[count];
+
+            workers[0] = new Worker("Antonov", "AA", "trainee", 2014);
+            workers[1] = new Worker("Vadimov", "DD", "junior", 2003);
+            workers[2] = new Worker("Ivanchenko", "TT", "middle", 1998);
+            workers[3] = new Worker("Abrasyan", "AB", "senior", 1995);
+            workers[4] = new Worker("Zuravko", "VD", "essential", 2001);
+
+            //Sort struct by surname
+            Array.Sort(workers, new Comparison<Worker>((c, d) => c.surname.CompareTo(d.surname)));
```

Comment:
It's not your code. It's a copypaste. Why do you need new Comparison?

Reply:
Yes, its copypaste, I wanted sort workers by field surname, but it does not affect the solution of the task. I remove this code.

Reply:
Why not just Array.Sort(workers, (c, d) => c.surname.CompareTo(d.surname)); ?